packages/shared/components/Illustration.svelte

```diff
@@ -71,6 +71,10 @@
             lightmode: 'staking/assembly-info.png',
             darkmode: 'staking/assembly-info.png',
         },
+        'staking-notification': {
+            lightmode: 'staking/staking-notification.png',
```

Comment:
Unless we specifically need the PNG file, we can probably use the [SVG one](https://github.com/iotaledger/firefly/blob/develop/packages/shared/assets/illustrations/staking/staking-notice.svg) that's already there.

Reply:
Ahh didn't realise we already had a similar popup, I have done some refactoring.